### PR TITLE
Update irs990.yaml

### PR DESCRIPTION
**Pull request by ma-ji**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: datasets/irs990.yaml [hunk 1]
```diff
@@ -51,4 +51,8 @@ DataAtWork:
       URL: https://www.grantmakers.io
       AuthorName: Chad Kruse
       AuthorURL: https://www.chadkruse.com/
+    - Title: npo_classifier
```
Comment (jflasher) [inline datasets/irs990.yaml:54]: Thanks @ma-ji! Could you just add a little bit more of a description to the `Title` field and then this should be good to merge in?

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 7d9aa9673c11db76251f5a350ebf26110330b560
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=9536b3c0-4ee8-48c2-a128-b0f9717b61d9%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: datasets/irs990.yaml [hunk 2]
```diff
@@ -51,4 +51,8 @@ DataAtWork:
       URL: https://www.grantmakers.io
       AuthorName: Chad Kruse
       AuthorURL: https://www.chadkruse.com/
+    - Title: npo_classifier: Automated coding using machine-learning and remapping the U.S. nonprofit sector
```
Comment (jflasher) [inline datasets/irs990.yaml:54]: You just need to add quotes around the title text.

File: datasets/irs990.yaml (hunk 1 above)
Comment (ma-ji) [reply datasets/irs990.yaml]: Sure thing, just did that!

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 68c5ccbbf7c1cd9656e1a83f297dc7ae1cbbf6f2
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=2364e27b-f805-45c1-98d3-3e9400f32819%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: datasets/irs990.yaml (hunk 2 above)
Comment (ma-ji) [reply datasets/irs990.yaml]: Should be good now.

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: bfc88395dbdf71d86d5a5f5ebb21dfab45178ab0
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=68e85703-20ef-4d26-b93f-7ff6d2780ecb%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**